### ISSUE TITLE
请问是否需要验证集

### ISSUE THREAD

**Issue by lin-lcx** [huangzhii/SALMON#2]
您好，我发现您只使用了训练集和测试集，并没有使用验证集。也就是说在测试测试集时，使用的model仅仅是训练num_epochs后的结果，请问是这样吗？
另一个问题是，您的代码中似乎只输出了每一折测试集的结果，那么最终的结果是否需要统计将所有测试集合在一起的CI？
感谢您的回答！

**Comment by huangzhii** [comment]
It's a good question. Unlike computer science paper which usually takes training - validation - testing scheme with k-fold cross-validation, SALMON is an explorational project, which aims for validating whether multi-omic data with feature upfront engineering can achieve equal or better survival prediction. So yes, it only has training and testing groups. In the future, it is also a good practice to include a validation set, and use that for tuning the hyper-parameters. However, that really depends on the limited amount of the data.
